assets/package.json

```diff
@@ -10,7 +10,7 @@
   "devDependencies": {
     "@editorjs/editorjs": "^2.28.0",
     "@editorjs/header": "^2.7.0",
-    "@editorjs/image": "^2.8.1",
+    "@editorjs/image": "github:martide/editor-js-image#WEB-8452-image-alt",
```

Comment:
@rcmonsayac this looks alright but why not open up a PR to the maintainers ?

Reply:
@dgross881 it actually is the issue, there is a PR already and I just copied it. Maintainers are not responding here https://github.com/editor-js/image/issues/30 since 2019

Reply:
PR in forked repo https://github.com/martide/editor-js-image/pull/1/files, will change this to point to master once merged